scaffolding-chef-infra/lib/windows/scaffolding.ps1

```diff
@@ -1,7 +1,15 @@
 #
 # A scaffolding for Chef Policyfile packages
 #
-
+if(!$scaffold_chef_client){
+    $scaffold_chef_client = "stuartpreston/chef-client"
```

Comment:
You will want to give these variables `SCRIPT` scope if they are to be used in other functions and have not been previously det with `SCRIPT` scope.